src/app/pages/add-network/add-network.tsx

```diff
@@ -49,7 +51,7 @@ export const AddNetwork = () => {
           setError('');
           try {
             const origin = new URL(url).origin;
-            const response = await fetchPrivate(`${origin}/v2/info`);
+            const response = await network.fetchFn(`${origin}/v2/info`);
```

Comment:
I noticed in the AMS open PR that @beguene is handling this differently, just as a heads up. With the upgrade, he created a `fetchPrivate` using `createFetchFn()` as a utility. I will need to add the new api key to this change at some point.